### DISCUSSION TITLE
Latency (Avg) showing 0 for dataset runs

### DISCUSSION THREAD

**Discussion by karthik-TLIT** [langfuse/langfuse#2137] [Support] [answered]
I'm using a self-hosted (docker) langfuse for evaluating my RAG pipeline.
The pipeline is mainly built using llama-index, and I'm following the code from the [datasets docs page](https://langfuse.com/docs/datasets/overview).

Even though the individual traces in the runs have a few seconds of latency - the avg. latency on runs page shows 0
![image](https://github.com/langfuse/langfuse/assets/166356832/7b55aff9-2ce1-4e59-b022-7eae138fa5f6)
Can someone suggest what could be the issue?

**Comment by marcklingen** [comment, marked as answer]
thanks for sharing, fixed here: https://github.com/langfuse/langfuse/pull/3411
